pkg/apis/zookeeper/v1beta1/zookeepercluster_types.go

```diff
@@ -188,8 +188,19 @@ func (s *ZookeeperClusterSpec) withDefaults(z *ZookeeperCluster) (changed bool)
 }
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-// ZookeeperCluster is the Schema for the zookeeperclusters API
 // +k8s:openapi-gen=true
```

Comment:
can we add a comment here specifying below makers comments are for generating crd using kubebuilder ( as we use operator-sdk framework for other things I feel it would be better to mention that)

Reply:
Done. Thanks for reviewing.